src/pages/wallet-overview/wallet-overview.scss

```diff
@@ -94,7 +94,7 @@ page-wallet-overview {
     }
 
     .funds-overview {
-        min-height: 313px;
+        min-height: 287px;
```

Comment:
That's awfully specific :P

Reply:
Just added a bit pixels so when the balance is loading the transfer button won't teleport up and down since the balance is not defined in the UI and therefor not showing. It was a bit too much pixels so when showing 2 balances the transfer button had some white space beneith it :P